homeassistant/components/climate/ephember.py

```diff
@@ -152,3 +185,33 @@ def max_temp(self):
     def update(self):
         """Get the latest data."""
         self._zone = self._ember.get_zone(self._zone_name)
+
+    @staticmethod
+    def map_mode_hass_eph(operation_mode):
+        """Map from home assistant mode to eph mode."""
+        from pyephember.pyephember import ZoneMode
+        if operation_mode == STATE_AUTO:
```

Comment:
You can instead make a dictionary that holds the map, put at the module level, and use `getattr` on `ZoneMode`here to get the correct attribute.